packages/boxel-ui/test-app/tests/integration/components/resizable-panel-group-horizontal-test.gts

```diff
@@ -0,0 +1,310 @@
+      testArgs;
+    this.lengthPx = lengthPx;
+    this.isHidden = isHidden;
+    this.defaultLengthFraction = defaultLengthFraction;
+    this.minLengthPx = minLengthPx;
+
+    this.showResizeHandle = showResizeHandle;
+
+    this.innerContainerStyle = innerContainerStyle;
+    this.outerContainerStyle = outerContainerStyle;
+  }
+}
+
+class RenderController {
+  @tracked containerStyle = '';
+  @tracked panels: PanelProperties[] = [];
+}
+
+interface MyTestContext extends RenderingTestContext {
+  renderController: RenderController;
+}
+
+module('Integration | ResizablePanelGroup | horizontal', function (hooks) {
+  setupRenderingTest(hooks);
+  hooks.beforeEach(function (this: MyTestContext) {
+    this.renderController = new RenderController();
+    this.renderController.panels = [
+      new PanelProperties(
+        { defaultLengthFraction: 0.6 },
+        {
+          showResizeHandle: true,
+          outerContainerStyle:
+            'border: 1px solid red; height: 100%; overflow-y:auto',
+        },
+      ),
+      new PanelProperties(
+        { defaultLengthFraction: 0.4, minLengthPx: 50 },
+        {
+          outerContainerStyle: 'border: 1px solid red; height: 100%',
+        },
+      ),
+    ];
+  });
+
+  async function renderHorizontalResizablePanelGroup(
+    renderController: RenderController,
+  ) {
+    await render(<template>
+      {{! template-lint-disable no-inline-styles }}
+      <div id='test-container' style={{renderController.containerStyle}}>
+        <ResizablePanelGroup
+          @orientation='horizontal'
+          @reverseCollapse={{true}}
+          as |ResizablePanel ResizeHandle|
+        >
+          {{#each renderController.panels as |panel index|}}
+
+            <ResizablePanel
+              @defaultLengthFraction={{panel.defaultLengthFraction}}
+              @lengthPx={{panel.lengthPx}}
+              @minLengthPx={{panel.minLengthPx}}
+              @isHidden={{panel.isHidden}}
+            >
+              {{#if (eq index 1)}}
+                <div
+                  class='panel-{{index}}-content'
+                  style={{panel.outerContainerStyle}}
+                >
+                  <div>
+                    Panel
+                    {{index}}
+                  </div>
+                </div>
+              {{else}}
+                <div
+                  class='panel-{{index}}-content'
+                  style={{panel.outerContainerStyle}}
+                >
+                  Panel 2
+                </div>
+              {{/if}}
+            </ResizablePanel>
+            {{#if panel.showResizeHandle}}
+              <ResizeHandle />
+            {{/if}}
+          {{/each}}
+        </ResizablePanelGroup>
+      </div>
+    </template>);
+    await sleep(100); // let didResizeModifier run
+  }
+
+  test<MyTestContext>('it can lay out panels horizontally', async function (assert) {
+    this.renderController.containerStyle =
+      'max-width: 100%; height: 100px; width: 318px;';
```

Comment:
The 18px surplus is to account for the resize handle right. Can we make that explicit and assert like the handle is 18 px

Reply:
Can we just used one fixed width 118px for all test too?  So, I ppl dont need to know why the width is changing for all test.

Reply:
hmm well when I change to 100 + 18

<img width="904" alt="✖ Boxel Tests 2024-06-20 14-24-57" src="https://github.com/cardstack/boxel/assets/43280/eceba9ee-00de-466f-9b3a-ae99e681e8c5">

because the second panel has a minimum width of 50px.

I’ll try a consistent 300 everywhere instead 🤞🏻